src/native/libs/System.IO.Ports.Native/pal_termios.c

```diff
@@ -437,22 +437,22 @@ int32_t SystemIoPortsNative_TermiosReset(intptr_t handle, int32_t speed, int32_t
 
     if (tcgetattr(fd, &term) < 0)
     {
-        return  -1;
+        return -1;
     }
 
 #if HAVE_CFMAKERAW
     cfmakeraw(&term);
 #else
-    term.c_iflag &= ~(IMAXBEL|IGNBRK|BRKINT|PARMRK|ISTRIP|INLCR|IGNCR|ICRNL|IXON);
+    term.c_iflag &= ~(IMAXBEL | BRKINT | PARMRK | ISTRIP | INLCR | IGNCR | ICRNL | IXON);
     term.c_oflag &= ~OPOST;
-    term.c_lflag &= ~(ECHO|ECHONL|ICANON|ISIG|IEXTEN);
-    term.c_cflag &= ~(CSIZE|PARENB);
+    term.c_lflag &= ~(ECHO | ECHONL | ICANON | ISIG | IEXTEN);
+    term.c_cflag &= ~(CSIZE | PARENB);
     term.c_cflag |= CS8;
 #endif
-    term.c_cflag |=  (CLOCAL | CREAD);
-    term.c_lflag &= ~((tcflag_t)(ICANON | ECHO | ECHOE | ECHOK | ECHONL | ISIG | IEXTEN ));
+    term.c_cflag |= (CLOCAL | CREAD);
+    term.c_lflag &= ~((tcflag_t)(ICANON | ECHO | ECHOE | ECHOK | ECHONL | ISIG | IEXTEN));
     term.c_oflag &= ~((tcflag_t)(OPOST));
-    term.c_iflag = IGNBRK;
+    term.c_iflag |= IGNBRK;
```

Comment:
This is the fix.

Reply:
Thanks for the quick reply. Note that the same flag is already set in the first line:

term.c_iflag &= ~(IMAXBEL|**IGNBRK**|BRKINT|...

Do you still think, this is the fix?

Reply:
I removed it from the first line.

Reply:
I can do. Just didn't liked that overloaded/long line at first glance. 😅

Reply:
Considering https://www.man7.org/linux/man-pages/man3/termios.3.html
```
	cfmakeraw() sets the terminal to something like the "raw" mode ...

	The terminal attributes are set as follows:

           termios_p->c_iflag &= ~(IGNBRK | BRKINT | PARMRK | ISTRIP | INLCR | IGNCR | ICRNL | IXON);
           termios_p->c_oflag &= ~OPOST;
           termios_p->c_lflag &= ~(ECHO | ECHONL | ICANON | ISIG | IEXTEN);
           termios_p->c_cflag &= ~(CSIZE | PARENB);
           termios_p->c_cflag |= CS8;
```